### ISSUE TITLE
"origami" and "rendering

### ISSUE THREAD

**Issue by z76531** [linyq2117/CLIP-ES#18]
May I ask if you have a code for the experiment related to the selection of prompt words "origami" and "rendering"? Can you provide it? Thank you, looking forward to your reply

**Comment by linyq2117** [comment]
We are sorry that we no longer keep the code for prompt selection. You can simply run the code using different prompts and compare the mIoU results. Alternatively, you can use the original CLIP to perform classification and record the logits of each image, then calculate sharpness for each prompt based on Equ(9) in our paper.

**Comment by linyq2117** [comment]
Closed for inactivity.
